crates/api_crud/src/community/remove.rs

```diff
@@ -44,7 +41,7 @@ pub async fn remove_community(
   .with_lemmy_type(LemmyErrorType::CouldntUpdateCommunity)?;
 
   // Mod tables
-  let expires = data.expires.map(naive_from_unix);
+  let expires = check_expire_time(data.expires)?;
```

Comment:
Should community removal really have an expiration? Doesnt make sense to me.

Reply:
Hrm... ya that looks like an error in the API. We don't currently have a `ban_expires` column on community at all.

Reply:
True, its only written to mod log. Gonna remove it.